cmd/generate.go

```diff
@@ -93,6 +93,11 @@ function create_cluster() {
   roachprod create "$CLUSTER" -n $NODES --lifetime "{{.Lifetime}}" --clouds "$CLOUD" \
     --$CLOUD-machine-type "{{.MachineType}}" {{.NodeEastLocation}} {{.EvaledArgs}} {{.UsEastAmi}} \
     --label {{.Usage}}
+  
+  echo "---- start of cpu info for cluster $CLUSTER ----"
+  roachprod run "$CLUSTER":1 -- cat /proc/cpuinfo
+  echo "---- end of epu info for cluster $CLUSTER ----"
+
```

Comment:
We probably don't need this? I thought that cpu info  output was added recently.

Reply:
This should be auto merged when I rebase master, but I will revert it.